### ISSUE TITLE
[defaultTimezone] not working

### ISSUE THREAD

**Issue by nwinesett** [kimai/kimai#1327]
#### Expected behaviour

1) In Admin Panel | Advanced, set America/New York for the field entitled: time zone for new users and customers 
2) Click Save and page reloads
3) Admin Panel | Customers | add customer
4) Dialog box opens but default customer is:  Africa/Abdijan

#### Actual behaviour

default customer is:  Africa/Abdijan

#### Steps to reproduce

See above

#### Reproducible on demo.kimai.org? [yes|no]

Not available anymore :(  ... I still love the speed and layout of v1

#### Kimai version [for example: dev 1.1.0.1389|github master|github develop]

v1.3.5.1400

#### Relevant log entries from log file (debug tab or temporary/logfile.txt)

None

**Comment by simonschaufi** [comment]
As kimai 1 is not developed further, I'm only doing payed support. If you want the bug to be fixed, please sponsor me.

**Comment by jimlucas** [comment]
I still prefer to use the older 1.3.x too.

**Comment by kevinpapst** [comment]
@jimlucas then please keep on contributing and keep it alive! I am sure there are quite a lot of installation of v1 out there. 

If you ever feel as you want to contribute some ideas how v2 could be improved, please open feature requests for it.

**Comment by nwinesett** [comment]
@jimlucas 
So cool!  I've copied your change to my installation.  I've tested it from both the admin panel and the timesheet lower lists and it works great in both scenarios.  Thanks for the commit.  I vote to update the master.
 
> I still prefer to use the older 1.3.x too.

@kevinpapst 
I've installed a test installation of v2 and tested it briefly on my site, but v1 is just so much quicker to use for me and everything is on one page.  I don't know v2 well enough to comment there, but I will keep it up to do an look at it from time to time.

I will tell you that in v1, my favorite feature is inline editing for the descriptions.  I can go through my work day without ever having to load a new page by simply starting a timer and selecting my radio buttons.  I've set up a customer named "New Client" and a project named "New Project" which serve as quick entries if somebody new calls.  I can just fill in a quick description and add the new customer and new project later.  No clicking or page loads to take me off my main screen.

I have some new good ideas (e.g. batch archiving - i.e. unchecking visibility) for v1 but I have to learn some coding to contribute better.  

Finally, this is long overdue, but a big thank you to the contributors of v1.  I know it's missing some features that you would get in a commercial product like review/approval workflow, but on the other hand, those commercial products don't touch the fast workflow in v1.

I'm new to github.  Would it be appropriate for me to mark this closed now?  Or should I wait until it's uploaded to master?
